src/auth.js

```diff
@@ -8,7 +8,8 @@ const createToken = (user) => {
     user,
     secret,
     {
-      expiresIn: '10d'
+      expiresIn: '10d',
+      sub: user.id
```

Comment:
I fear this won't work. You have to use the string `subject` (ref: https://www.npmjs.com/package/jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback).

Reply:
Fixed